hysterical-horses/code_jam/users/apps.py

```diff
@@ -0,0 +1,9 @@
+from django.apps import AppConfig
+
+
+class UsersConfig(AppConfig):
+    name = "users"
+
+    def ready(self):
+        import users.signals
+        print(users.signals.__name__)
```

Comment:
This print should be removed as well, but I see why you did this - to pass linting tests. However, you should have just added a `# NOQA: F401` to ignore the unused import error.
```suggestion
        import users.signals  # NOQA: F401
```